conan/cli/command.py

```diff
@@ -120,6 +120,12 @@ def parse_args(self, args=None, namespace=None):
             self._conan_api.config.global_conf.update_conf_definition(confs)
         return args
 
+    def format_help(self):
+        formatted_help = super().format_help()
+        formatted_help += ("\nNOTE: Use always positional parameters first\n"
+                           "      conan <command> <positional args> [optional args]\n")
+        return formatted_help
```

Comment:
```suggestion
        lines = formatted_help.splitlines()
        note = ("\nNOTE: Use always positional parameters first\n"
                "      conan <command> <positional args> [optional args]\n")
        lines.insert(lines.index(""), note)
        return "\n".join(lines)
```

I tried it, but I was not convinced, as it shows at the very end, not very visible when there are arguments. 
Then I tried this, it seems it would be a bit better IMHO.

Still, it is also missing in the short usage error when some arg is missing or incorrect. This can be solved by adding the ``format_usage()`` method too.

All in all, seems a bit too dirty/too much for a known Python bug: https://github.com/python/cpython/issues/53584

Not sure what would be the best approach to be honest.

Reply:
Let's just make sure we mention it somewhere in the docs then!